ormconfig.ts

```diff
@@ -0,0 +1,44 @@
+import { ConnectionOptions } from 'typeorm';
+import {
+  TYPEORM_HOST,
+  TYPEORM_USERNAME,
+  TYPEORM_PASSWORD,
+  TYPEORM_DATABASE,
+  TYPEORM_PORT,
+  TYPEORM_SYNCHRONIZE,
+  TYPEORM_LOGGING,
+  TYPEORM_MIGRATIONS_RUN,
+  PRODUCTION_ENV
+} from './src/config';
+
+const config: ConnectionOptions = {
+  type: 'postgres',
```

Comment:
Shouldn't this be TYPEFORM_TYPE?

Reply:
Yes, but actually this is throwing an error since it's expecting a specific type `"postgres" || "mysql" ...` and I haven't found a proper way of casting this from the env variables... we can create a function inside this file to cast it into one of those specific types or leave this as "hard coded" as it is... In the midtime, I'll add this as a note on this PR

![Screen Shot 2021-03-12 at 11 58 14](https://user-images.githubusercontent.com/8084303/110957492-6b7ab100-832a-11eb-808a-6e18c0fe9822.png)

Reply:
Oh yes, the undefined, maybe casting to string? You can check before if you want, either way it's going to throw an error.

Reply:
The issue is deeper than checking `undefined`. It's like `"postgres"` or `"mysql"` were defined as types, so receiving a string is not enough

![Screen Shot 2021-03-12 at 12 09 05](https://user-images.githubusercontent.com/8084303/110958743-c82a9b80-832b-11eb-9b56-a9bb8d33b833.png)

Reply:
Cool. Ughh, ok, yes, the library should have a type defined for that, but seems not. An alternative is to create the same type but in our side, and cast it to that type.

Reply:
Yes!! I was checking we can have something like this with "supported" or most commonly used DBs (at least for us)

![Screen Shot 2021-03-12 at 12 20 18](https://user-images.githubusercontent.com/8084303/110960246-4fc4da00-832d-11eb-8fa9-5aad6cf80d3d.png)

Reply:
Done! 👍